dts/bindings/net/wireless/nordic,nrf-radio.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Nordic DFE antennas matrix description
+
+    dfegpio[n]-gpios properties provide GPIO pins to Radio peripheral. Provided
+    GPIO pins are used to drive antennas switching by Radio Direction Finding
+    Extension. E.g. to use 12 antennas 4 GPIO pins must be provided,
+    4 GPIO pins support switching up to 16 antennas.
+
+    All dfegpio[n]-gpios properties have required property set to false.
+    It depends on end user which Radio PSEL_GPIO[n] registers will be used
+    to drive antenna switches.
+    Nevertheless there is a requirement that at least two antennas are
+    available. That means at least one dfegpio[n]-gpios property must be
+    provided, to give Radio possibility to switch between these two antennas.
+
+compatible: "nordic,nrf-radio"
+
+include: [base.yaml]
+
+properties:
+    ant-num:
```

Comment:
I think this should be renamed to `dfe-ant-num`, to indicate it's just for the number of switchable antennas for the direction finding extension.

Reply:
I've changed the dfe_ant to proposed radio node. I've verified interrupt, register address and size. They are correct.
Also ant-num is now dfe-ant-num.